### DISCUSSION TITLE
Information about the upcoming version 1.17

### DISCUSSION THREAD

**Discussion by alexcimander** [Digitaler-Impfnachweis/covpass-android#172] [Announcements]
📢 **Information about the upcoming version 1.17** 📢  
Version 1.17 of the CovPass and CovPassCheck is planned to be released early february. It will include a variety of features and fixes that were requested by the community.

**Planned Main Features CovPass:**
•	Improvements on Trustlist-Updates: The apps will now use etag, gzip and other http-features to improve the update-mechanisms of the trustlist.

**Planned Main Features CovPassCheck:**
•	The CovPassCheck now offers scan-sequences to check your 2G+ Status. The awaited booster-recognition, especially regarding the use-case “valid test not necessary after booster vaccination” is planned to be added with the next release 1.18.

**Planned Bug-Fixes:**
•	Multiple UI-Bugfixes as Accessibility-Enhancements are planned for the release.  
  
Thank you all for your feedback, ideas and critics!

stay safe and best regards,

Team CovPass and CovPassCheck ✔️

**Comment by treysis** [comment]
Early Februar...wow...just wow 🙄

**Reply by boecko** [reply]
In Februar no one will care about this app anymore. Just look at the recent reviews https://play.google.com/store/apps/details?id=de.rki.covpass.checkapp&showAllReviews=true

**Reply by alexcimander** [reply]
Hey @treysis, 

I completly understand your frustration about the delay between "new regulations within Germany" and "scan possibilities within the CovPassCheck". But I also kindly ask you to be patient with us since with all the new regulations, data-privacy concerns and changing admission concepts on federal-state-level our team is working nonstop for a feasible and usable approach for all citizens.

We thank all of you for your patience but also ideas, critics and support! 1.17 and the following releases are focused on the admission concepts to further simplify check-scenarios.

Thank you again! 👍

**Comment by r41d** [comment]
I'm wondering how exactly checking 2G+ is different from the "valid test not necessary after booster vaccination" criterion.
Isn't this what fulfilling 2G+ (by being boostered, which is the most common case) basically is?

**Reply by fefrei** [reply]
So far, this feature is about recognizing 2G+ state by scanning both a vaccination certificate and a test certificate of the same person.
This has parity with what the CWA currently does: If the CWA attests you have 2G+ status (which it so far only does if you have a vaccination and test), it will now show two certificates, scanning both of which should then convince the CovPassCheck app of the 2G+ status as well.

Note: I'm not related to the project, this is just my reading of the notes above.

**Comment by herryilmaz** [comment]
When(if?) the issue for the people who had previously Corona will be solved by the App? I only had to be vaccinated once and received directly 2/2 for the number of the doses on my vaccination certificate. I also got my booster vaccination (technically second) but also on these it's written 2/2 so the app doesn't see this as a "Booster" vaccination, but rather just a second one.

I'm of course not the only one with this issue: https://www.br.de/nachrichten/netzwelt/app-chaos-genesene-muessen-sich-nicht-dreimal-impfen-lassen,SuCw9JI

Is it ever going to be solved?!
